### ISSUE TITLE
NTR: Isovalerylglycine present in urine (Lab: chemistry: organic acids)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8584]
New term request Isovalerylglycine present in urine (segal_030509173005):

**Comment by pnrobinson** [comment]
@MickeySegal
are we referring to [N-isovalerylglycine](https://www.ebi.ac.uk/ols/ontologies/chebi/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FCHEBI_70984)? Please provide context

**Comment by MickeySegal** [comment]
https://omim.org/entry/243500 and https://www.ncbi.nlm.nih.gov/books/NBK558236/ and https://pubmed.ncbi.nlm.nih.gov/17712735/ say just "Isovalerylglycine"

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000602
- label: Elevated urinary isovalerylglycine level
- parents: [Abnormal urinary acylglycine profile (HP:0012073)](https://hpo.jax.org/app/browse/term/HP:0012073)
- pmid: PMID:17850781
- def: The amount of isovalerylglycine in the urine, normalized for urine concentration, is above the upper limit of normal.
- comment: Isovalerylglycine (IVG) is an acyl glycine. The excretion of certain IVG is increased in isovaleric acidemia.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8584
- synonyms:
